operators/pkg/controller/elasticsearch/sset/getter.go

```diff
@@ -15,3 +17,20 @@ func Replicas(statefulSet appsv1.StatefulSet) int32 {
 	}
 	return 0
 }
+
+// UpdatePartition returns the updateStrategy.Partition index, or falls back to the number of replicas if not set.
+func UpdatePartition(statefulSet appsv1.StatefulSet) int32 {
```

Comment:
Why rename?

Reply:
To better match the other function in this file: `Replicas()`. So we have:

```go
replicas := sset.Replicas(s)
updatePartition := sset.UpdatePartition(s)
```

Do you think `GetReplicas` and `GetUpdatePartition` would be better?

Reply:
My only complain is that I read update as a verb and it feels surprising it doesn't update anything :) Partition? But this reads as a verb too... CurrentPartition?

Reply:
Agreed, this is confusing.
Maybe using `Get` for both is better.